pkg/tests/tests.go

```diff
@@ -84,6 +84,14 @@ var tests = []helpers.TestOptions{
 		Method:   http.MethodPost,
 		Handler:  handlers.TestClusterAuthorizations,
 	},
+	{
+		TestName:    "self-terms-review",
+		Path:        "/api/authorizations/v1/self_terms_review",
+		Method:      http.MethodPost,
+		Body:        handlers.EmptyBody(),
```

Comment:
I think you can just not assign it and it will do the same, most of the other request work like that.

Reply:
Good point! I did this in haste. Let me remove this.